findimports.py

```diff
@@ -559,13 +562,16 @@ def __init__(self):
         self._module_cache = {}
         self._warned_about = set()
         self._stderr = sys.stderr
-        self._exts = ('.py', '.so', '.dll')
+        self._exts = ['.py', '.so', '.dll']
         if hasattr(sys, '_multiarch'):  # pragma: nocover
             # Ubuntu 14.04 LTS renames
             # /usr/lib/python2.7/lib-dynload/datetime.so to
             # /usr/lib/python2.7/lib-dynload/datetime.x86_64-linux-gnu.so
             # (https://github.com/mgedmin/findimports/issues/3)
-            self._exts += f".{sys._multiarch}.so"
+            self._exts.append(f".{sys._multiarch}.so")
+        if hasattr(sys, 'platform'):
+            ver = ''.join(map(str, sys.version_info[:2]))
+            self._exts.append(f'.cpython-{ver}-{sys.platform}.so')
```

Comment:
I don't think this is right.  `sys.platform` is `"linux"`, but all the modules in /usr/lib/python3.12/lib-dynload/ use "x86_64-linux-gnu" in this spot on my machine.

You want `sysconfig.get_config_var('EXT_SUFFIX')` here.

(Personally, I don't even remember what `self._exts` is used for.  Discovering whether named Python modules exist on disk?)

Reply:
It is a fallback for determining if it is a module.